src/platform/ESP32/bluedroid/BLEManagerImpl.cpp

```diff
@@ -1149,9 +1146,12 @@ BLEManagerImpl::CHIPoBLEConState * BLEManagerImpl::GetConnectionState(uint16_t c
     {
         if (freeIndex < kMaxConnections)
         {
-            memset(&mCons[freeIndex], 0, sizeof(CHIPoBLEConState));
-            mCons[freeIndex].Allocated = 1;
-            mCons[freeIndex].ConId     = conId;
+            mCons[freeIndex].PendingIndBuf = nullptr;
```

Comment:
I wonder whether `CHIPoBLEConState` should grow a `Reset` function...  Followup perhaps?

Reply:
Added to TODO list. Every Impl has its own version…